### ISSUE TITLE
Parameter validation failed: Invalid type for parameter parameters[4].value.stringValue, value: 1942-02-19, type: <class 'datetime.date'>, valid types: <class 'str'>

### ISSUE THREAD

**Issue by revmischa** [chanzuckerberg/aurora-data-api#5]
Code is here: https://github.com/jetbridge/sls-flask/blob/master/TEMPLATE/db/fixtures.py and https://github.com/jetbridge/sls-flask/blob/master/TEMPLATE/model/user.py

Trying to insert a row into serverless postgres that includes a date column.

```
Traceback (most recent call last):
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/bin/flask", line 10, in <module>
    sys.exit(main())
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/flask/cli.py", line 966, in main
    cli.main(prog_name="python -m flask" if as_module else None)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/flask/cli.py", line 586, in main
    return super(FlaskGroup, self).main(*args, **kwargs)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/flask/cli.py", line 426, in decorator
    return __ctx.invoke(f, *args, **kwargs)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/cyber/dev/jb/slsflask/TEMPLATE/commands.py", line 19, in seed_db_cmd
    seed_db()
  File "/Users/cyber/dev/jb/slsflask/TEMPLATE/db/fixtures.py", line 30, in seed_db
    db.session.commit()
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/scoping.py", line 162, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/aws_xray_sdk/ext/sqlalchemy/util/decorators.py", line 61, in wrapper
    res = func(*args, **kw)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1027, in commit
    self.transaction.commit()
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 494, in commit
    self._prepare_impl()
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 473, in _prepare_impl
    self.session.flush()
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/aws_xray_sdk/ext/sqlalchemy/util/decorators.py", line 61, in wrapper
    res = func(*args, **kw)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2470, in flush
    self._flush(objects)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2608, in _flush
    transaction.rollback(_capture_exception=True)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 68, in __exit__
    compat.reraise(exc_type, exc_value, exc_tb)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 153, in reraise
    raise value
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2568, in _flush
    flush_context.execute()
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
    rec.execute(self)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 589, in execute
    uow,
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 245, in save_obj
    insert,
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 1137, in _emit_insert_statements
    statement, params
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 982, in execute
    return meth(self, multiparams, params)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1101, in _execute_clauseelement
    distilled_params,
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1250, in _execute_context
    e, statement, parameters, cursor, context
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1478, in _handle_dbapi_exception
    util.reraise(*exc_info)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 153, in reraise
    raise value
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1246, in _execute_context
    cursor, statement, parameters, context
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 581, in do_execute
    cursor.execute(statement, parameters)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/aurora_data_api/__init__.py", line 197, in execute
    res = self._client.execute_statement(**execute_statement_args)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/botocore/client.py", line 634, in _make_api_call
    api_params, operation_model, context=request_context)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/botocore/client.py", line 682, in _convert_to_request_dict
    api_params, operation_model)
  File "/Users/cyber/.virtualenvs/slsflask-IkGl8I07/lib/python3.7/site-packages/botocore/validate.py", line 297, in serialize_to_request
    raise ParamValidationError(report=report.generate_report())
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter parameters[4].value.stringValue, value: 1942-02-19, type: <class 'datetime.date'>, valid types: <class 'str'>
```

**Comment by kislyuk** [comment]
This should be fixed in v0.2.0.

**Comment by olinger** [comment]
@kislyuk Was this fix only for postgres? I am seeing this same issue in 0.2.0 with serverless mySQL 

"botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter parameters[1].value.stringValue, value: 2020-02-01 15:16:45.252348, type: &lt;class 'datetime.datetime'&gt;, valid types: &lt;class 'str'&gt;"

**Comment by revmischa** [comment]
Also happening for timedelta :/

**Comment by revmischa** [comment]
We're trying to use an actual `datetime.date` as a column value on insert/update. ADA is passing it to boto3 as a date not a string, which I think is causing the error. Same with timedelta, UUID, etc.
Possible fix would be something similar to #11
